packages/core/package.json

```diff
@@ -41,6 +41,7 @@
     "clean": "npx rimraf ./lib"
   },
   "dependencies": {
+    "@ceramicnetwork/anchor-utils": "^1.1.0-rc.0",
```

Comment:
Could it be set in `devDependencies` if the package is only used in tests please?